### ISSUE TITLE
generated schedule event is out of date

### ISSUE THREAD

**Issue by gotson** [awslabs/aws-sam-cli#373]
It seems the `schedule` event sent by CloudWatch and the one generated by `0.2.11` are different.

Here is what `sam local` yields:
```
 ~  sam local generate-event schedule                                                                    Sat Apr 28 16:56:45 2018
{
  "account": "123456789012",
  "region": "us-east-1",
  "detail": {},
  "detail-type": "Scheduled Event",
  "source": "aws.events",
  "time": "1970-01-01T00:00:00Z",
  "id": "cdc73f9d-aea9-11e3-9d5a-835b769c0d9c",
  "resources": [
    "arn:aws:events:us-east-1:123456789012:rule/my-schedule"
  ]
}
```

Here is what i see today in my console when i want to generate a new rule:
![screengrab](https://i.imgur.com/WQ8Vj7i.png)

The `version` field is missing in the `sam local` version.

**Comment by sanathkr** [comment]
Can you send a pull request to fix this? Should be pretty simple one

**Comment by gotson** [comment]
The problem i have with that is consistency. Basically this project is not linked at all with the rest of AWS, so any future change on AWS event types will require an update of the SAM CLI.

Isn't there any template or swagger available on AWS Services that provide the structure of events ?
Hooking SAM CLI into that would align the data, ensure consistency and prevent future breaks.

**Comment by jfuss** [comment]
@gotson We hear you on the consistency but this is the reality at the moment. We have nothing to hook into, so the best we can do is keep things updated and release more often. 

This was released as apart of [v0.4.0](https://github.com/awslabs/aws-sam-cli/releases/tag/v0.4.0), therefore closing
